### PR TITLE
fix pic url

### PR DESCRIPTION
**Pull request by yangyuliufeng**
### Ⅰ. Describe what this PR does
add the pic in the doc of adding new runtime

### Ⅱ. Does this pull request fix one issue?

#484 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: docs/zh/dev/runtime_dev_guide.md [hunk 1]
```diff
@@ -7,38 +7,38 @@ Fluid提供了Runtime接口，并且假设Runtime和Dataset是一对一的关系
 
 Dataset的生命周期流程如图所示：
 
-![dataset](/Users/qiulingwei/Downloads/dataset.png)
+![dataset](https://github.com/fluid-cloudnative/fluid/blob/master/static/dataset_lifecycle.png)
```
Comment (xieydd) [inline docs/zh/dev/runtime_dev_guide.md:10]: How about relative paths?

**Review by xieydd — CHANGES_REQUESTED** [review]

File: docs/zh/dev/runtime_dev_guide.md (hunk 1 above)
Comment (yangyuliufeng) [reply docs/zh/dev/runtime_dev_guide.md:10]: @xieydd 更改到了../../media/images

Comment (xieydd) [reply docs/zh/dev/runtime_dev_guide.md:10]: @yangyuliufeng Have not seen changed?

**Review by xieydd — APPROVED** [review]
/lgtm

**Review by cheyang — APPROVED** [review]

**Pull request merged**